water/.gitignore

```diff
@@ -0,0 +1,13 @@
+.idea/
+.vscode/
+.pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
+node_modules/
+dist/
+yarn-error.log
+.env
```

Comment:
you can do subdirectory-specific .gitignores? o.O

if not, you'll wanna move this up to the root `./.gitignore`